File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String input = Integer.toString(x);
        int n = String.valueOf(x).length() - 1;
        for(int i = 0; i < n; i++){
            if(input.charAt(i) != input.charAt(n - i)){
                return false;
            }
        }
        return true;
    }
}

File: 9-palindrome-number/palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string input = x.ToString();
        int n = input.Length - 1;
        for (int i = 0; i <= n / 2; i++) {
            if (input[i] != input[n - i]) {
                return false;
            }
        }
        return true;
    }
}